### PR TITLE
dynamic create node version info

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature
**What this PR does / why we need it**:

now when we exec `kubectl get node`
```
kubectl get node
NAME                 STATUS     ROLES    AGE    VERSION
172.20.11.45-cloud   Ready      master   11d    v1.15.3
192.168.4.27-pi1     NotReady   edge     5d1h   v1.15.0-kubeedge-v1.0.0

```
the edgenode version is `v1.15.0-kubeedge-v1.0.0`, it is from the config of edge.yaml of edgecore

```
edged:
    register-node-namespace: default
    hostname-override: 192.168.4.27-pi1
    ...
    version: v1.15.0-kubeedge-v1.0.0
    runtime-type: docker
    remote-runtime-endpoint: unix:///var/run/dockershim.sock
    ...

```

it is hardcode.

this pr is dynamic get edge version and register it to k8s.

this is an example:
```
kubectl get node
NAME                 STATUS   ROLES    AGE    VERSION
172.20.11.45-cloud   Ready    master   11d    v1.15.3
192.168.4.27-pi1     Ready    edge     5d1h   v1.15.3-kubeedge-v1.0.0-344+2444224ba56784

```

because now we use k8s v1.15.3 library.  so we use v1.15.3 to stand for support k8s 1.15.3 version.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: common/constants/default.go [hunk 1]
```diff
@@ -74,4 +74,6 @@ const (
 	CSIOperationTypeControllerPublishVolume   = "controllerpublishvolume"
 	CSIOperationTypeControllerUnpublishVolume = "controllerunpublishvolume"
 	CSISyncMsgRespTimeout                     = 1 * time.Minute
+
+	CurrentSupportK8sVersion = "v1.15.0"
```
Comment (fisherxu) [inline common/constants/default.go]: Seems v1.15.3 now

Comment (kadisi) [reply common/constants/default.go]: ok

**Comment by fisherxu** [comment]
/lgtm
Thanks @kadisi

**Pull request merged**